packages/core/src/Modal/index.tsx

```diff
@@ -156,7 +162,8 @@ export const Modal: FC<ModalProps> = ({
         focusTrapOptions={{
           initialFocus: focusDialog ? `#${id}` : undefined,
           escapeDeactivates: false, // We use our own stack
-          clickOutsideDeactivates: true, // 😱😱😱 We need this to prevent click capturing
+          clickOutsideDeactivates: allowOutsideClick, // 😱😱😱 We need this to prevent click capturing
+          allowOutsideClick, // This is allows to turn on click capturing to prevent tabbing outside the component
```

Comment:
According to the information of the prop, if `clickOutsideDeactivates` is `true`, `allowOutsideClick` option is **ignored**. So why do you need the `allowOutsideClick` option?

Reply:
Removed option of allowOutsideClick.